src/wavesurfer.js

```diff
@@ -200,8 +200,8 @@ var WaveSurfer = {
         if (paused) {
             this.params.scrollParent = false;
         }
+        this.drawer.progress(progress);
         this.backend.seekTo(progress * this.getDuration());
-        this.drawer.progress(this.backend.getPlayedPercents());
```

Comment:
Again, an unrelated and risky change. The backend is supposed to be the data source for the drawer.